Gruntfile.js

```diff
@@ -24,7 +24,7 @@ module.exports = function(grunt) {
         dist: {
             options: {
                 mangle: true,
-                compress: true
+                compress: {}

```

Comment:
I was getting the error reported here https://github.com/gruntjs/grunt-contrib-uglify/issues/287 <https://github.com/gruntjs/grunt-contrib-uglify/issues/287>. They suggested using compress:{} instead of compress:true. Although now I look at the page fresh I see they also suggest updating to a newer grunt. I’ll test it when I’m back in office on Wednesday and revert this change if compress:true is fine

> On 10 Aug 2015, at 09:38, Nicholas Hallahan <notifications@github.com> wrote:
> 
> In Gruntfile.js <https://github.com/hallahan/LeafletPlayback/pull/34#discussion_r36611150>:
> 
> > @@ -24,7 +24,7 @@ module.exports = function(grunt) {
> >          dist: {
> >              options: {
> >                  mangle: true,
> > -                compress: true
> > +                compress: {}
> What does replacing true with an empty object do?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/hallahan/LeafletPlayback/pull/34/files#r36611150>.
>